bbmcmann/307MovieProject
# New voting

OK, so this is most of the implementation for voting. 
What I changed: 
-review schema and rout functionality to reflect voting arrays of user id's instead of vote counts
-added logic to check if user has voted on a review on render
-change review doc when a user votes by adding and removing them from the correct lists
-added styling for voting indication
-fixed rendering errors
-changed review data  formatting to match the db
Right now, it is set up to use a test ID representing the current logged in user. I will need to use the user cookies from a future merge to get this to work for the signed in user.

**_To test:_**
go to the Cars movie page. One review (should be the first) should have some upvotes and downvotes, and one of the vote arrows will be large and yellow. This indicates the user's selected vote. You can click the other arrow to see the arrow switch along with the numbers. You can see this in the DB as well by filtering reviews by movie_id: 920. it should also be open of the first reviews, you can look at the arrays to see them change. You can also console.log it to see the differences
